testsuite/features/support/constants.rb

```diff
@@ -967,6 +981,10 @@
         sle-manager-tools-for-micro5-pool-x86_64-5.5
         sle-manager-tools-for-micro5-updates-x86_64-5.5
       ],
+    'sl-micro-6.0' => # TODO: Add  sl-manager-tools-for-micro6-pool-x86_64-6.0   sl-manager-tools-for-micro6-updates-x86_64-6.0 when they are ready
+      %w[
+        sl-micro-6.0-pool-x86_64
```

Comment:
This is missing in the `TIMEOUT_BY_CHANNEL_NAME` list below.

Reply:
Kostas ask me if you don't know how to determine it

Reply:
How do you know how to determine the values? Do we have this documented somewhere? Maybe a comment above the list would be nice to have it in one place.

Reply:
I added them, let me know if they are wrong values, I pretty much copied them from the other SLE micro versions